packages/app/src/app/pages/common/Modals/PickSandboxModal/index.js

```diff
@@ -0,0 +1,95 @@
+import React, { Component } from 'react';
+import { inject, observer } from 'mobx-react';
+
+import Button from 'app/components/Button';
+import Row from 'common/components/flex/Row';
+import Input, { TextArea } from 'common/components/Input';
+
+import {
+  Container,
+  Heading,
+  Explanation,
+} from '../LiveSessionEndedModal/elements';
+
+import { Field, Label } from './elements';
+
+class PickSandboxModal extends Component {
+  state = {
+    title: this.props.store.explore.pickedSandboxDetails.title || '',
+    description:
+      this.props.store.explore.pickedSandboxDetails.description || '',
+  };
+
+  onChange = e =>
+    this.setState({
+      [e.target.name]: e.target.value,
+    });
+
+  render() {
+    const { signals, store } = this.props;
+    const { id } = store.explore.pickedSandboxDetails;
+    const { title, description } = this.state;
+
+    return (
+      <Container>
+        <Heading>Pick this sandbox</Heading>
+        <Explanation>
+          Please add a title and description to this sandbox if none exists or
+          you think you have a better description for it
```

Comment:
Maybe also mention that we will show the given title/description on the explore page.